lm_eval/tasks/math.py

```diff
@@ -0,0 +1,343 @@
+import json
+import os
+from lm_eval.utils import sh
+from lm_eval.metrics import mean
+from lm_eval.base import Task, rf
+import abc
+
+class Math(Task):
```

Comment:
class MATH(Task):
'''
This dataset is based on the following paper: https://arxiv.org/abs/2103.03874
'''